### PR TITLE
Cookbook: Recuerdos de los Rancheros

### PR DESCRIPTION
**Pull request by Yokubaitis**
If time allows, please advise to let me know if I did the ALT TEXT correctly since this was the only thing which I was unsure of.

### CODE REVIEW

**Comment by kirschbombe** [comment]
Hi. Your alt-text looks great! There are a few things to work on before we merge into the main repository:
1. The <relatedItem> title (line 23) should be "UCLA Library Special Collections"
2. In the bibliography, the last citation doesn't seem complete - is there a publication title?
3. Each of your images needs a title - the <head> element
4. The attribution statement should include the creator, date, and institution: "Mary Johnson, c1894. Image courtesy of UCLA Library Special Collections."
Let me know if questions.

File: articles/cookbook.xml [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+         </titleStmt>
+         <publicationStmt>
+            <publisher>Colleen Jaurretche</publisher>
+            <date when="2015">Fall 2015</date>
+         </publicationStmt>
+         <notesStmt>
+            <relatedItem>
+               <bibl>
+                  <author></author>
+                  <title>Rare Southern California Manuscript Cookbook, UCLA Library Special Collections</title>
```
Comment (kirschbombe) [inline articles/cookbook.xml]: Remove "Rare Southern California Manuscript Cookbook, "

File: articles/cookbook.xml [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+            <p>An original cookbook containing authentic hispanic recipes.  Thought to be one of the first cookbooks, it includes watercolor depictions of various Latin American foods.</p>
+         </note>
+         
+         <div type="bibliography">
+            <listBibl>
+               <bibl>Pilcher, Jeffrey M.. “Tamales or Timbales: Cuisine and the Formation of Mexican National Identity, 1821-1911”. The Americas 53.2 (1996): 193–216.</bibl>
+               <bibl>"California Ranchos." California Ranchos. Toucan Valley Publications, Inc., n.d.</bibl>
+               <bibl>Reid, Hiram Alvin, and Alfred James McClatchie. "History of Pasadena." Google Books.
+Pasadena History Company. (1895): 201.</bibl>
+               <bibl>"State Normal School Groups." (1892): 5-9.</bibl>
```
Comment (kirschbombe) [inline articles/cookbook.xml]: Is the citation here complete?

File: articles/cookbook.xml [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+            </listBibl>
+         </div>
+         
+         <div type="images">
+            <figure xml:id="fig1">
+               <graphic url="../images/Cookbook1.jpg"/>
+               <head></head>
```
Comment (kirschbombe) [inline articles/cookbook.xml]: need image titles here

File: articles/cookbook.xml [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+               <note type="attribution">San Pascuel: A Ranchero House, Image courtesy of UCLA Library Special Collections</note>
```
Comment (kirschbombe) [inline articles/cookbook.xml]: Attribution should include the name of the creator (Mary Johnson) and the date (c1894), then "Image courtesy of...)

**Comment by Yokubaitis** [comment]
Thank you for bringing those problems to my attention! I believe I have made the following changes correctly.  Please advise me if I have not.  Also, do I need to submit another pull request?

**Comment by kirschbombe** [comment]
it looks like you are including the image title with the image attribution - no need to repeat the title twice. Just use "San Pascuel: A Ranchero House" for the title, then "Mary E. Johnston, c1894. Image courtesy of UCLA Library Special Collections." for the attribution statement. Follow that for the rest of the images.

Also, I'm still not clear on what the final citation is - is this a book or an article? a website? Can you point me to the resource so I can help you format the citation? We just want people to be able to find it!

**Comment by Yokubaitis** [comment]
Sorry for my late reply.  I have made the changes requested.  It was a group member who made the previous citation.  She sent me the link and I have remade it to the best of my ability but I am not sure it is correct.  Here is the link to the source:  http://www.registrar.ucla.edu/archive/catalog/1892-93catalog.pdf

**Comment by kirschbombe** [comment]
Here's the proper format following MLA: Tenth Annual Catalogue of the State Normal School, at Los Angeles, for the School Year Ending June 23, 1892, and Circular for 1892-92. Sacramento: State Office, 1892. http://www.registrar.ucla.edu/archive/catalog/1892-93catalog.pdf

Everything else looks great! Once you've edited the citation, go ahead and submit a new pull request and I'll merge your article into the main project page.

**Comment by Yokubaitis** [comment]
I tried submitting a new pull request and it just sent me back to this one.  How do I create a new one?

**Comment by kirschbombe** [comment]
you can close this one, then start a new one

**Pull request closed without merging**